pymc3/step_methods/hmc/base_hmc.py

```diff
@@ -201,9 +201,9 @@ def astep(self, q0):
         stats = {
             "tune": self.tune,
             "diverging": bool(hmc_step.divergence_info),
-            "perf_counter_diff_ns": perf_end - perf_start,
-            "process_time_diff_ns": process_end - process_start,
-            "perf_counter_ns": perf_end,
+            "perf_counter_diff": perf_end - perf_start,
+            "process_time_diff": process_end - process_start,
+            "perf_counter_start": perf_start,
```

Comment:
Same comment on the name, and out of curiosity: why did switch from `perf_end` to `perf_start`?

Reply:
Again, I think the choice it kind of arbitrary. We can reconstruct the other one since we have the difference.
I just thought it might be a bit more intuitive to have the _start_ of the draw as an absolute value.